stacks/environment-local/lead.tf

```diff
@@ -111,3 +109,11 @@ module "lab_partner" {
   lab_partner_version         = var.lab_partner_version
 }
 
+module "prometheus-operator" {
+  source = "../../modules/tools/prometheus"
```

Comment:
did `terragrunt plan` not fail here?  I'd expect it to since it has moved locations.

can you fix this and verify that `terragrunt plan` works with these changes?

Reply:
Yup, just tested this branch against sandbox. Caught this issue and pushed up the fix